.travis.yml

```diff
@@ -8,24 +8,25 @@ php:
 env:
   global:
     - COMPOSER_ARGS="--prefer-dist --no-interaction --no-suggest"
+    - PHPUNIT_ARGS=""
 
 matrix:
   include:
     - php: 7.1
       env:
         - DEPS=low
-        - CS=true
```

Comment:
@GawainLynch wdyt? we only need to collect once right? given the changeset is the same on all runs. Kept high/low matrix for this.

Reply:
![image](https://user-images.githubusercontent.com/1047696/34444656-e60c84ba-eccf-11e7-93d7-377b83a4cd10.png)

Reply:
WFM :+1: